src/modules/endpoint/__tests__/DatabaseConnector.spec.js

```diff
@@ -0,0 +1,225 @@
+// @flow
+
+import DatabaseConnector from '../DatabaseConnector'
+import DatabaseContext from '../DatabaseContext'
+import moment from 'moment-timezone'
+import RNFetchBlob from '../../../__mocks__/rn-fetch-blob'
+import CityModelBuilder from '../../../testing/builder/CitiyModelBuilder'
+import CategoriesMapModelBuilder from '../../../testing/builder/CategoriesMapModelBuilder'
+import LanguageModelBuilder from '../../../testing/builder/LanguageModelBuilder'
+import EventModelBuilder from '../../../testing/builder/EventModelBuilder'
+
+jest.mock('rn-fetch-blob')
+const databaseConnector = new DatabaseConnector()
+
+beforeEach(() => {
+  RNFetchBlob.fs._reset()
+})
+
+describe('DatabaseConnector', () => {
```

Comment:
I still think testing where the `storeXyz` functions write their JSON files would be pretty helpful.

Reply:
I've now added some tests for the store functions